sktime/markov/tools/estimation/sparse/_bindings/include/mle_trev_sparse.h

```diff
@@ -0,0 +1,214 @@
+//
+// Created by mho on 8/6/20.
```

Comment:
i am not really looking at the C stuff

Reply:
i'm skimming it but don't trust me

Reply:
ok

Reply:
sure no worries, it is just a port of existing cython/c extensions to c++ to make things a bit more maintainable and tidier.